### ISSUE TITLE
Yogi watch?

### ISSUE THREAD

**Issue by F21** [yui/yogi#60]
Shifter has a watch command which is very useful in dev: I can just save my file and go to my browser and reload.

Yogi seems to be missing a watch command probably because building/testing/generating docs takes much longer than just building.

However, it would still be nice if we can ask yogi to watch our `src` folder with different file types: `--types js css docs` etc. This would ask yogi to watch all js, css and docs.

**Comment by F21** [comment]
Scratch this. It turns out that we can use `yogi build --watch`.

However, this is missing from `help --detailed`. So, my suggestion is to add `--watch` under the help documentation for `build` :)

**Comment by davglass** [comment]
Does `yogi build --watch`  not work for you?

**Comment by davglass** [comment]
Sorry, the GH app is slow. All `--` commands to `yogi build` should pass down to `shifter`.

**Comment by F21** [comment]
Could you make that more apparent in `help --detailed`? As a beginner, my starting point was using `help --detailed`, and it is not apparent that all commands will be passed down to shifter :)

**Comment by davglass** [comment]
I'll look into it.

**Comment by klamping** [comment]
Just to throw in my two cents, I also ran into this lack of knowledge with using 'yogi build'. I wasn't aware that shifter has a --watch param, so the documentation on passing params to shifter didn't really help. I could have gone down that path to find it out, but I think it's a common enough use case that it should be more apparent.  I submitted a PR (https://github.com/yui/yogi/pull/64) with an idea for how to be more explicit with this.
